chrome/content/zotero/xpcom/zotero.js

```diff
@@ -1095,6 +1095,13 @@ Services.scriptloader.loadSubScript("resource://zotero/polyfill.js");
 	 */
 	this.launchFile = function (file) {
 		file = Zotero.File.pathToFile(file);
+
+		var customPDFReaderPath  = Zotero.Prefs.get("extensions.zotero.pdf.launcher");
+		if(file.path.toUpperCase().endsWith(".PDF") && customPDFReaderPath){
```

Comment:
- Move `customPDFReaderPath` first in the conditional
- `toLowerCase()` and `.pdf`
- Space after `if` and before `{`}

The bigger issue is that technically the extension isn't what we want to use here — we want to use `attachmentContentType` on the `Zotero.Item` and look for `application/pdf`. For now we should probably just do that test in `ZoteroPane.viewAttachment()` (where we do various other tests to figure out what to do) and call `Zotero.launchFileWithGivenApplication()` from there. Down the line we can extract that parsing logic out, but right now that's the only place that calls `launchFile()` where this is actually necessary.